fall18/p2/README.md

```diff
@@ -1,3 +1,147 @@
 # Project 2
 
-More details coming soon.
+In this project, you'll learn about creating and calling functions, arguments, return values and math module. To start, download these files (Note: right click on the links below, not the links above that only show a preview):
+
+* [main.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p2/main.py)
+* [project.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p2/project.py)
+* [test.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p2/test.py)
+* [area.csv](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p2/area.csv)
+* [population.csv](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p2/population.csv)
+
+You will change main.py and hand it in. You should not change test.py and project.py, and you should not hand them in; it's only purpose is to tell you your grade in advance.
+
+After you've downloaded all the above 5 files to the same directory, open your terminal, navigate to that directory, and run the following:
+
+```
+python test.py
+```
+
+You should see the following output:
+(Note: the python version may be different depending on the version of the python interpreter you use.)
+
+```
+Your Python version: Python 3.6.5
+
+Running your program with this command: python3 main.py
+
+RESULTS:
+  Problem 1: function getMaximumLand not found
+  Problem 2: function getMinimumPopulationDensity not found
+  Problem 3: function predictPopulation not found
+  Problem 4: function predictPopulation not found
+  Problem 5: function calcGrowthRate not found
+  Problem 11: PASS
+  Problem 12: expected (5686986.0) but found (0)
+  Problem 13: expected (86935.83) but found (0)
+  Problem 14: expected (52.002450206414075) but found (0)
+  Problem 15: expected (796039951.1495125) but found (0)
+  Problem 16: expected (0.005853103209551789) but found (0)
```

Comment:
Why not add back some of your original tests to get a nice even 20?

Reply:
Ok.